src/test/resources/examples/resultset/valid-resultset-example.json

```diff
@@ -1,35 +1,35 @@
 {
   "pipelineInformation": {
     "name": "pipeline_info",
-    "path": "./",
+    "path": "./pipeline_info",
     "children": [],
     "softwareVersions": {
       "name": "software_versions",
       "fileType": "csv",
-      "path": "pipeline_info/software_versions"
+      "path": "./pipeline_info/software_versions.csv"
     },
     "pipelineReport": {
       "name": "pipeline_report",
       "fileType": "txt",
-      "path": "pipeline_info/pipeline_report"
+      "path": "./pipeline_info/pipeline_report.txt"
     },
     "executionReport": {
       "name": "execution_report",
       "fileType": "txt",
-      "path": "pipeline_info/execution_report"
+      "path": "./pipeline_info/execution_report.txt"
     }
   },
   "qualityControl": {
     "name": "multiqc",
-    "path": "./",
+    "path": "./multiqc",
     "children": [
       {
         "name": "star_salmon",
-        "path": "multiqc/star_salmon",
+        "path": "./multiqc/star_salmon",
         "children": [
           {
             "name": "multiqc_report.html",
-            "path": "multiqc/star_salmon/multiqc_report.html",
+            "path": "./multiqc/star_salmon/multiqc_report.html",
```

Comment:
Even though it might not be so relevant for the example, the changes that I made reflects how most pipelines are structured. Subfolders there are a special case of the RNAseq pipelines and maybe a couple more.

Reply:
true, but its a valid example, and that is what this json represents. In the schema we defined `qualityControl` as a `DataFolder`, so it can be arbitrarily nested with DataFiles and DataFolders. The only thing we can check for validation is, that it has `multiqc` in the name of the top level multiqc folder.

Reply:
ok sure then it is fine to leave it like this, just to mention that that would be a more common example 👍